torch/_dynamo/eval_frame.py

```diff
@@ -203,7 +203,9 @@ def eager_then_compile(*args, **kwargs):
                         )
                     )
 
-                compiler_fn = callback._torchdynamo_orig_callable._torchdynamo_orig_callable.compiler_fn
+                compiler_fn = (
+                    callback._torchdynamo_orig_callable._torchdynamo_orig_callable
```

Comment:
I realized that this is much more robust since sometimes `compiler_fn` doesn't exist, but you always can just invoke the callable directly.